config.yaml

```diff
@@ -39,10 +39,10 @@
             # Make sure that the samples you merge have the same sample features (e.g paired, sense, dup_type)
```

Comment:
Not explain which samples are "merged", see previous comment.

"sample features" should rather be "samples are of the same kind"